client/js/views/playbar.react.jsx

```diff
@@ -160,13 +173,13 @@ var Playbar = React.createClass({
           <ButtonToolbar>
             <ButtonGroup bsSize='medium'>
               <OverlayTrigger placement='top' delayShow={500} overlay={<Tooltip> rewind </Tooltip>}>
-                <Button onClick={this.rewind}><Glyphicon glyph='backward' /></Button>
+                <Button onClick={this.rewind} style={this.getStyle('rewind')}><Glyphicon glyph='backward' /></Button>
```

Comment:
seems a bit roundabout, will investigate other ways but maybe needed due to react